Code/Mantid/Framework/API/src/FrameworkManager.cpp

```diff
@@ -113,6 +132,20 @@ void FrameworkManagerImpl::UpdateInstrumentDefinitions() {
   }
 }
 
+
+/// Update instrument definitions from github
```

Comment:
This comments looks like a copy&paste. Instrument definition => new available versions?